Workstream Policy.md

```diff
@@ -318,6 +318,21 @@ case-by-case basis by the [Steering Group].
 [Other Publications] published by the WHATWG bear text to be determined by the
 [Steering Group].
 
+### Notice for Translations
+
+WHATWG [Living Standards][Living Standard] are under a permissive license that allows
+independent translations. WHATWG recommends the following notice for translations, including the link:
```

Comment:
Here and below, "the WHATWG" when used as a noun

Reply:
Sounds good, also Translations should be lowercase for consistency.